pkg/controller/virtinformers.go

```diff
@@ -58,6 +58,7 @@ import (
 
 const (
 	OperatorLabel = kubev1.ManagedByLabel + "=" + kubev1.ManagedByLabelOperatorValue
+	errMsg = "unexpected object"
```

Comment:
I suggest you define the error and not the string.

How about: `var unexpectedObjectError = errors.New("unexpected object")`

Reply:
Thanks for the advice, Done.